web/src/agents/node/get_last_sequence.rs

```diff
@@ -0,0 +1,41 @@
+use at2_node::client::{self, Client};
+use at2_ns::ThinUser;
+use wasm_bindgen_futures::spawn_local;
+use yew::worker::*;
+
+use crate::config::Config;
+
+/// Get the last sequence for a user
+pub struct GetLastSequence {
+    link: AgentLink<Self>,
+    client: Client,
+}
+
+impl Agent for GetLastSequence {
+    type Reach = Context<Self>;
+    type Message = (HandlerId, Result<sieve::Sequence, client::Error>);
+    type Input = ThinUser;
+    type Output = Result<sieve::Sequence, client::Error>;
+
+    fn create(link: AgentLink<Self>) -> Self {
```

Comment:
FoMyRuK: why is it `create` and not `new`?

Reply:
there isn't constructor per say in Rust; function returning `Self` can be used to construct object but there aren't default.
`new` is the common way to name it. This `create` function is in the `Agent` trait of "yew", and goes with its semantic (there is a `destroy` function, components can get recreated on page change, ...)